src/views/Onboarding/Welcome.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div class="screen">
+    <div class="welcome-container">
+      <div class="content">
+        <h1 class="welcome-title">
+          Welcome to {{ productName }}
+        </h1>
+        <p class="welcome-description">
+          Congratulations for downloading {{ productName }}! You are just a <strong>few minutes</strong> away from getting your service mesh fully online.
```

Comment:
```suggestion
          Congratulations on downloading {{ productName }}! You are just a <strong>few minutes</strong> away from getting your service mesh fully online.
```

(and also in the snapshot)